frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -18,27 +18,55 @@ const getPropertyListUrl = (filter: IPropertyFilter) =>
 const getPropertyReportUrl = (filter: IPropertyFilter) =>
   `${ENVIRONMENT.apiUrl}/reports/properties?${filter ? queryString.stringify(filter) : ''}`;
 
-const filter: IPropertyFilter = {
+const defaultFilter: IPropertyFilter = {
   page: 1,
-  quantity: 50,
+  quantity: 10,
   agencies: [1, 2, 3, 4, 5, 6],
 };
 
 const PropertyListView: React.FC = () => {
   const columns = useMemo(() => cols, []);
+
+  // We'll start our table without any data
   const [data, setData] = useState<IProperty[]>([]);
+  const [filter, setFilter] = useState<IPropertyFilter>({ ...defaultFilter });
+  // const [loading, setLoading] = useState(false);
+  const [pageCount, setPageCount] = useState(0);
+  const fetchIdRef = useRef(0);
+
+  // This will get called when the table needs new data
+  const fetchData = useCallback(
```

Comment:
Using a callback instead of an effect. The table will call this when it needs new data.